scripts/fast-reboot

```diff
@@ -202,34 +202,6 @@ function request_pre_shutdown()
     fi
 }
 
-function recover_issu_bank_file()
```

Comment:
I understand that the paths in the current check are not relevant as the issu bank file format has changed.

However, I am not clear on why we do not change file names to use new format, rather than remove the check altogether. 
From our discussion - this issue is not seen so far on newer branches, but is that a guaranee that the issue will never be seen again? If yes, then how is that guaranteed?

What is the downside of keeping this change w/ new file format?

Reply:
> However, I am not clear on why we do not change file names to use new format, rather than remove the check altogether. From our discussion - this issue is not seen so far on newer branches, but is that a guaranee that the issue will never be seen again? If yes, then how is that guaranteed?

As far as I remember the issu_bank.txt got corrupted due to unknown reasons. Even with this check I don't see a guarantee issu_bank.txt won't get corrupted after the check and recovery.

See the original PR - https://github.com/sonic-net/sonic-utilities/pull/1466, the way to reproduce it is modify warm-reboot script to artificially corrupt it. 
 
> What is the downside of keeping this change w/ new file format?

We, in SONiC, have to track changes SDK makes to it's files. And there is not just a single file, there are more. Not all of them can be restored from SONiC. It could also happen that SDK changes the format of the file and SONiC overwrites it to an old format leading to a failed warm boot.


> If you are concerned about the auto-fix ultimately breaking something, then do you rather want to detect a missing/broken bank file, and abort warmboot (do the check before `set +e`)?

I would then ask SDK to do the check. But then, the logic they have to implement is - write the content and read it to make sure it is written. Seems redundant. Again, does not guarantee files corruption afterwards.

Reply:
> I would then ask SDK to do the check. But then, the logic they have to implement is - write the content and read it to make sure it is written. Seems redundant. Again, does not guarantee files corruption afterwards.

I did not follow this part completely. IMO, this is not redundant if it works in some situations and detect corrupted files to avoid warmboot. This has happened before in older images, and can potentially happen now too. Aborting warm reboot will prevent any risk of dataplane downtime.